content_scripts/hud.coffee

```diff
@@ -114,6 +113,11 @@ HUD =
   unfocusIfFocused: ->
     document.activeElement.blur() if document.activeElement == @hudUI?.iframeElement
 
+  clearFocus: ->
+    if @hudUI?.showing
```

Comment:
I didn't look hard, but Is this `@hudUI?` conditional necessary? Is it possible to be exiting find mode without the @hudUI being initialized?

Reply:
The `?` seems necessary on slow pages. When a page is loading slowly, press <kbd>/</kbd> will call `showFindMode` to init and show HUD iframe on `documentComplete`, so if a user click/press <kbd>Esc</kbd> before `documentComplete`, it will exit before `HUD.hudUI` gets ready.